addons/sale_management/models/sale_order_template_line.py

```diff
@@ -105,7 +96,7 @@ def _prepare_order_line_values(self):
         self.ensure_one()
         return {
             'display_type': self.display_type,
-            'name': self.name,
+            'name': self.name or self.product_id.get_product_multiline_description_sale(),
```

Comment:
I'd prefer if you delegate the name computation to the standard compute when possible:
```python
vals = {}
if self.name:
    vals['name'] = self.name
return vals